### PR TITLE
JAR-6148

### PR DESCRIPTION
**Pull request by oxedions**
Dirty but works.
FM v2, based on FM v1 image.

### CODE REVIEW

File: V2/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,12 @@ RUN chmod -R 777 /usr /etc /run /var; \
...
     echo 'https://%PUBLICADDR%:5902/owncloud/index.php/login?user=%NIMBIXUSER%&password=%RANDOM64%' > /etc/NAE/url.txt
 
+FROM scratch
+
+COPY --from=V1 / /
+
 COPY owncloud-start.sh /usr/local/bin/owncloud-start.sh
 COPY AppDef.json /etc/NAE/AppDef.json
+
+RUN mkdir -p /etc/NAE && touch /etc/NAE/{url.txt,AppDef.json}
```
Comment (nimbixler) [inline V2/Dockerfile]: that's not sufficient to speed up the pull - the syntax is as follows:
`RUN mkdir -p /etc/NAE && touch /etc/NAE/screenshot.png /etc/NAE/screenshot.txt /etc/NAE/license.txt /etc/NAE/AppDef.json
`
re: https://jarvice.readthedocs.io/en/latest/docker/#best-practices

**Review by nimbixler — CHANGES_REQUESTED** [review]
please see comments about final layer in Dockerfile

File: V2/Dockerfile (hunk 1 above)
Comment (oxedions) [reply V2/Dockerfile]: Ok, I was not sure if I could do that if not these files were present. Fixing.

**Review by nimbixler — APPROVED** [review]
ship it

**Pull request merged**